README.md

```diff
@@ -221,6 +222,21 @@ result = ArtVandelay::Import.new(:users).csv(csv_string, attributes: {email_addr
 # => #<ArtVandelay::Import::Result>
 ```
 
+##### Stripping whitespace
+
+```ruby
+csv_string = CSV.generate do |csv|
+  csv << ["email_address  ", " passcode  "]
+  csv << ["  george@vandelay_industries.com  ", "  bosco  "]
+end
+
+result = ArtVandelay::Import.new(:users, strip: true).csv(csv_string, attributes: {email_address: :email, passcode: :password})
+# => #<ArtVandelay::Import::Result>
+
+result.rows_accepted
+# => [{:row=>["george@vandelay_industries.com", "bosco"], :id=>1}]
+```
```

Comment:
@stevepolitodesign is it preferred that I use an example for the simple case, and not the custom attributes case, just for simplicity?

Reply:
Thank you for asking. I think you should keep the existing custom attributes example, since it highlights that this feature will strip empty spaced from both the headers and values.